Monash-FIT3170/JRVS
# Feature/no ref/edit-node-info

- All lesson/quiz/video edit pages now can edit the title and heading/desc
- when the title and heading/desc is edited, the node title and desc is also changed in the unit tree

To test:

You can use this admin account for the editing functionality:
username: **adminprofile**
password: **password**

- go into the admin account
- append/insert new node of any lesson type
- edit, and go back and check node title/desc and lesson title/desc
